block-producer/src/server/api.rs

```diff
@@ -46,7 +46,7 @@ where
         request: tonic::Request<SubmitProvenTransactionRequest>,
     ) -> Result<tonic::Response<SubmitProvenTransactionResponse>, Status> {
         let request = request.into_inner();
-        debug!(target: COMPONENT, tx = ?request.transaction);
+        debug!(target: COMPONENT, ?request);
```

Comment:
If we logging the entire request here. I wonder if there is much value to logging the proof on line 66 below.

Reply:
@bobbinth, on line 49 we log request in binary form (`Vec<u8>`), but on line 66 it's deserialized and more readable. I thing, we should keep both.